src/layout/VerifyEmailWarning.js

```diff
@@ -0,0 +1,50 @@
+import { useDataQuery, useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { NoticeBox } from '@dhis2/ui'
+import React from 'react'
+
+const getUserProfileQuery = {
+    me: {
+        resource: 'me',
+    },
+}
+const getSystemSettingsQuery = {
+    systemSettings: {
+        resource: 'systemSettings',
+    },
+}
+
+const VerifyEmailWarning = () => {
+    const { data: userData, error: userError } =
+        useDataQuery(getUserProfileQuery)
+    const { data: systemData, error: systemError } = useDataQuery(
+        getSystemSettingsQuery
+    )
+
+    const errorAlert = useAlert(({ message }) => message, { critical: true })
+
+    const enforceVerifiedEmail =
+        systemData?.systemSettings?.enforceVerifiedEmail
+    const emailVerified = userData?.me?.emailVerified
+
+    if (userError || systemError) {
```

Comment:
I think this would be better as a NoticeBox if we're going to show an error about not retrieving information (e.g. like we have elsewhere in the app: https://github.com/dhis2/user-profile-app/blob/e9207810974328e0a8ded774988b73c5b9a760cd/src/personalAccessTokens/generateTokenModal/AuthoritiesWarning.component.js#L27-L39).

However, I think if these requests were going to fail, the app seems like it wouldn't have loaded (https://github.com/dhis2/user-profile-app/blob/e9207810974328e0a8ded774988b73c5b9a760cd/src/AppWrapper.js#L71-L73), so maybe will not be relevant after refactoring.